tests/transactional/transactional_update.pm

```diff
@@ -116,10 +119,17 @@ sub run {
     check_reboot_changes;
     check_package;
 
+    record_info 'Continue', 'Continue modifying an snapshot -snapshots #5 and #6';
+    trup_call "pkg install" . rpmver('feature');
+    trup_call "--continue pkg install" . rpmver('optional');
```

Comment:
I don't find the `--continue` parameter of transactional-update, so I don't know what it does.

Reply:
it is documented here: https://github.com/openSUSE/transactional-update/blob/b2e191275b70d4c4c0e717498a914dcb156121db/sbin/transactional-update.in#L149
basically allows to continue modifying some previous snapshot without rebooting, check https://jira.suse.com/browse/SLE-7373

Reply:
So, is it new for Leap 15.2/SLE15-SP2?

Reply:
tw and sle15-sp2 is where was tested, I guess if they share the base code, it will be in Leap 15.2 but not our focus.

Reply:
It may be not your focus, but please do not break other tests ;)

Reply:
good point, I have to run it for Leap, I missed that was executed there.

Reply:
Maybe QAM also uses it. This was added for SLE15-SP1 or earlier.

Reply:
Added verification runs for Leap as well.